spec/models/article_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe "associations" do
+    it { should belong_to(:user) }
```

Comment:
it { should belong_to :user }

Reply:
sao ko dùng is_expected.to :v, thấy Rspec khuyến khích dùng is_expected.to